### PR TITLE
Implement fuh-ad measure and add oms validations

### PR DESCRIPTION
**Pull request by 13bfrancis**
## Purpose

This will complete the following measure: FUH-AD

#### Linked Issues to Close

https://qmacbis.atlassian.net/browse/OY2-8943

#### Pull Request Creator Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [x] This PR meets all acceptance criteria for those issues.
- [x] This PR and linked issue(s) are adequately documented
- [x] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [x] Someone has been assigned this PR.
- [x] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

File: services/ui-src/src/measures/2021/FUHAD/validation.ts [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+  const dateRange = data["DateRange"];
+  const deviationArray = getDeviationNDRArray(
+    data.DeviationOptions,
+    data.Deviations,
+    true
+  );
+  const didCalculationsDeviate = data["DidCalculationsDeviate"] === DC.YES;
+
+  let errorArray: any[] = [];
+  if (data["DidReport"] === "no") {
```
Comment (jdinh8124) [inline services/ui-src/src/measures/2021/FUHAD/validation.ts:117]: Could use the data constants for a few datas here:

  const whyNotReporting = data[DC.WHY_ARE_YOU_NOT_REPORTING];
  const OPM = data[DC.OPM_RATES];
  const dateRange = data[DC.DATE_RANGE];

  if (data[DC.DID_REPORT] === DC.NO) {

File: services/ui-src/src/measures/2021/FUHAD/validation.ts [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+import { getPerfMeasureRateArray } from "../../globalValidations";
+import { FormData } from "./types";
+import {
+  OmsValidationCallback,
+  omsValidations,
+  validateDenominatorGreaterThanNumerator,
+  validateOneRateLessThanOther,
+  validateRateNotZero,
+  validateRateZero,
+} from "measures/globalValidations/omsValidationsLib";
```
Comment (jdinh8124) [inline services/ui-src/src/measures/2021/FUHAD/validation.ts:24]: Ben's been trying to steer us towards using the 

import * as GV from "measures/globalValidations";

FVAAD has an example of this

**Review by jdinh8124 — APPROVED** [review]
A few comments but more so suggestions

**Comment by abelzer212** [comment]
"Follow-up" should be capitalized to "Follow-Up"

**Comment by abelzer212** [comment]
Figma Mismatch: Should be "Other" Not "Administrative Data Other"
Screen Shot 2022-03-22 at 10 51 26 AM
Screen Shot 2022-03-22 at 10 51 14 AM

File: services/ui-src/src/measures/2021/FUHAD/validation.ts (hunk 2 above)
Comment (charmcitygavin) [reply services/ui-src/src/measures/2021/FUHAD/validation.ts:24]: Is this captured anywhere? AMR-AD imports global validations like the code above, not using `import * as GV from "measures/globalValidations";` I wasn't aware that there was a different (and preferred) way to do it. Is there a ticket to go through the measures and make sure this is consistent? cc: @benjaminpaige

File: services/ui-src/src/measures/2021/FUHAD/validation.ts (hunk 1 above)
Comment (charmcitygavin) [reply services/ui-src/src/measures/2021/FUHAD/validation.ts:117]: This is cool! I didn't know about the data constants file. My questions for this is the same as above: is there a way to circulate this kinds of things to everyone so we can use them? AMR-AD is in Quality now but doesn't use these data constants (and it probably should). And is there already a plan to do this consistently across the existing measures?

File: services/ui-src/src/measures/2021/FUHAD/validation.ts (hunk 2 above)
Comment (13bfrancis) [reply services/ui-src/src/measures/2021/FUHAD/validation.ts]: Consider it done

**Review by abelzer212 — DISMISSED** [review]

**Review by abelzer212 — DISMISSED** [review]

**Review by benjaminpaige — DISMISSED** [review]

**Pull request merged**